packages/block-editor/src/store/reducer.js

```diff
@@ -2085,6 +2085,25 @@ export function hoveredBlockClientId( state = false, action ) {
 	return state;
 }
 
+/**
+ * Reducer setting zoom out state.
+ *
+ * @param {boolean} state  Current state.
+ * @param {Object}  action Dispatched action.
+ *
+ * @return {boolean} Updated state.
+ */
+export function zoomLevel( state = 100, action ) {
+	switch ( action.type ) {
+		case 'SET_ZOOM_OUT':
+			return action.zoom;
+		case 'RESET_ZOOM':
+			return 100;
+	}
+
+	return state;
+}
+
```

Comment:
Let's make the nomenclature consistent. Is it "zoom-out" or "zoom-level"? `type` and naming need to be consistent.

Reply:
Zoom out is anytime zoom level is less than 1.

Reply:
It's usually best to avoid using floating point numbers in JS. I went for `100` to mirror the traditional zoom scale of 0-100.

Is there a reason why you are suggesting `1`? Is it used somewhere else already?